All project code/api-example-responses/meteomatics/meteo-example.json

```diff
@@ -0,0 +1,106 @@
+{
+    "version": "3.0",
+    "user": "user",
+    "dateGenerated": "2021-09-20T09:01:10Z",
+    "status": "OK",
+    "data": [
+        {
+            "parameter": "t_2m:C",
+            "coordinates": [
+                {
+                    "lat": 50.23,
+                    "lon": 10.23,
+                    "dates": [
+                        {
+                            "date": "2021-09-20T00:00:00Z",
+                            "value": 10
+                        },
+                        {
+                            "date": "2021-09-20T01:00:00Z",
+                            "value": 9
+                        },
+                        {
```

Comment:
Are these just test cases or are they a part of actually functioning code?

Reply:
Just example responses, to be referenced when generating user-facing output in the search results. These are inaccessible to the user anyway (docker isn't reading these files) from the folder I put them in, so I don't think it should be a problem.